applications/FluidDynamicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -136,6 +139,15 @@ void AddCustomProcessesToPython(pybind11::module& m)
     .def("ComputeFlowOverBoundary", &MassConservationCheckProcess::ComputeFlowOverBoundary)
     ;
 
+    py::class_<ShockDetectionProcess, ShockDetectionProcess::Pointer, Process>
+    (m, "ShockDetectionProcess")
+    .def(py::init < ModelPart& >())
+    .def(py::init < ModelPart&, const Variable<double>& >())
+    .def("Initialize", &ShockDetectionProcess::ExecuteInitialize)
```

Comment:
It is a process, so you should use the functions of the process, not add new ones to the API
(i mean, otherwise it would be an utility)

Reply:
I started the implementation as a utility. However I thought this is something in between... I can change the API to make it in a more process-fashion.

Reply:
Done.